portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -53,9 +56,10 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     List<Comment> comments = new ArrayList<>();
 
     int commentLimit = getCommentLimit(request);
+    String languageCode = request.getParameter("lang");
```

Comment:
We'll probably want a default fallback in case this isn't set and you get `null` back from this call.

Reply:
Done.